From b2f1c7260c5f21ae52cb35b69b8f8d67428b5ce9 Mon Sep 17 00:00:00 2001
From: Ralf Gommers <ralf.gommers@gmail.com>
Date: Thu, 23 Dec 2021 18:28:54 +0100
Subject: [PATCH 3/3] TST: slightly bump test tolerance for a new lobpcg test

This was failing in the Meson build due to slightly different
build flags:
```
scipy/sparse/linalg/_eigen/lobpcg/tests/test_lobpcg.py:318: in test_tolerance_float32
    assert_allclose(eigvals, -np.arange(1, 1 + m), atol=1e-5)
E   AssertionError:
E   Not equal to tolerance rtol=1e-07, atol=1e-05
E
E   Mismatched elements: 1 / 3 (33.3%)
E   Max absolute difference: 1.31130219e-05
E   Max relative difference: 4.37100728e-06
E    x: array([-1.      , -2.000001, -2.999987], dtype=float32)
E    y: array([-1, -2, -3])
```
---
 scipy/sparse/linalg/_eigen/lobpcg/tests/test_lobpcg.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scipy/sparse/linalg/_eigen/lobpcg/tests/test_lobpcg.py b/scipy/sparse/linalg/_eigen/lobpcg/tests/test_lobpcg.py
index d42653f15..bc513803a 100644
--- a/scipy/sparse/linalg/_eigen/lobpcg/tests/test_lobpcg.py
+++ b/scipy/sparse/linalg/_eigen/lobpcg/tests/test_lobpcg.py
@@ -315,7 +315,7 @@ def test_tolerance_float32():
     X = rnd.standard_normal((n, m))
     X = X.astype(np.float32)
     eigvals, _ = lobpcg(A, X, tol=1e-5, maxiter=50, verbosityLevel=0)
-    assert_allclose(eigvals, -np.arange(1, 1 + m), atol=1e-5)
+    assert_allclose(eigvals, -np.arange(1, 1 + m), atol=1.5e-5)
 
 
 def test_random_initial_float32():
-- 
2.32.0.windows.2

